netbox/core/views.py

```diff
@@ -570,8 +570,9 @@ def get(self, request):
             return response
 
         # Serialize any CustomValidator classes
-        if hasattr(config, 'CUSTOM_VALIDATORS') and config.CUSTOM_VALIDATORS:
-            config.CUSTOM_VALIDATORS = json.dumps(config.CUSTOM_VALIDATORS, cls=ConfigJSONEncoder, indent=4)
+        for attr in ['CUSTOM_VALIDATORS', 'PROTECTION_RULES']:
+            if hasattr(config, attr) and getattr(config, attr):
```

Comment:
Just have to handle the default condition here.
```suggestion
            if hasattr(config, attr) and getattr(config, attr, None):
```

Reply:
I'll knock this out real quick if you don't mind, so we can slip it into the v4.2.2 release today.